src/main/java/com/project/controller/AdminCommentController.java

```diff
@@ -0,0 +1,80 @@
+package com.project.controller;
+
+import com.project.exception.AdminCommentNotFoundException;
+import com.project.exception.ApplicationNotFoundException;
+import com.project.exception.UserNotFoundException;
+import com.project.model.AdminComment;
+import com.project.model.request.RequestComment;
+import com.project.model.response.CommentResponse;
+import com.project.repository.ApplicationRepository;
+import com.project.service.AdminCommentService;
+import com.project.service.ApplicationService;
+import com.project.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping("api/comments")
+public class AdminCommentController {
```

Comment:
If I am not mistaken your API name is `"api/comments"` to fetch comment related resources. I would maybe suggest structuring your path to the comment resource like this: `"/api/applications/{applicationId}/comments"` because your comments are part of your application. When consuming your API it will be easier to understand where the comments come from.

Reply:
changed path to `"/api/applications/{appId}/comments"`, the controller now works for specific application and not all comments